Adds new pages to tables chapter.

katjacksonWB opened the pull request:
## Description

Adds a "create tables" page based on the text already in the reference section of the docs, with some edits for language and accuracy.

## Ticket

[DOCS-803](https://wandb.atlassian.net/browse/DOCS-803)

## Checklist

Check if your PR fulfills the following requirements. Put an `X` in the boxes that apply.

- [x] Files I edited were previewed on my local development server with `yarn start`. My changes did not break the local preview.
- [x] Build (`yarn docusaurus build`) was run locally and successfully without errors or warnings.
- [x] I merged the latest changes from `main` into my feature branch before submitting this PR.


[DOCS-803]: https://wandb.atlassian.net/browse/DOCS-803?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

katjacksonWB committed:
Adds "create tables page".

katjacksonWB committed:
Added section headers, removed test statments.

ngrayluna commented on docs/guides/tables/create-tables.md:
```diff
@@ -0,0 +1,81 @@
+
+## Table Visualizer
+
+Tables added directly to runs as above will produce a corresponding [Table Visualizer](visualize-tables.md) in the
```
Please add "./" to explicitly specify to look in this directory.

katjacksonWB committed:
Merge branch 'main' into katjackson-tables

katjacksonWB committed:
Vale + Comment Edits

johndmulhausen commented:
TODO: Remove `ja` pages 
TODO: @ngrayluna review, address feedback from #516 
TODO: Find SME/owner to review

johndmulhausen committed:
Merge branch 'main' into katjackson-tables

katjacksonWB committed:
Adds new pages to tables chapter. (#516)

## Description

Adds two new pages to the tables chapter that describe creating and
managing tables.

## Ticket

- Closes [DOCS-803](https://wandb.atlassian.net/browse/DOCS-803)
- Related to DOCS-814 (doesn't close)

## Checklist

Check if your PR fulfills the following requirements. Put an `X` in the
boxes that apply.

- [x] Files I edited were previewed on my local development server with
`yarn start`. My changes did not break the local preview.
- [x] Build (`yarn docusaurus build`) was run locally and successfully
without errors or warnings.
- [x] I merged the latest changes from `main` into my feature branch
before submitting this PR.


[DOCS-803]:
https://wandb.atlassian.net/browse/DOCS-803?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

---------

Co-authored-by: Noah Luna <15202580+ngrayluna@users.noreply.github.com>
Co-authored-by: John Mulhausen <john.mulhausen@wandb.com>

johndmulhausen committed:
Merge branch 'main' into katjackson-tables

johndmulhausen committed:
Merge branch 'main' into katjackson-tables

johndmulhausen commented:
Several issues afoot with this PR, unfortunately. Both "Create Table" and "Table Create" pages are being proposed. Style violations and myriad Vale errors. Blending in of `ja` and `en` content changes. And the original author has left the company. Strongly advise we close.

johndmulhausen commented:
Closing as we're going to move in a different direction with the tables docs
